.gitignore

```diff
@@ -1,6 +1,6 @@
 # Ignore automatically generated files.
 venv
-.coverage
+.coverage*
```

Comment:
How can I reproduce the creation of these files?

I was not aware that apart from `.coverage`, there are more `.coverage*` files being created. I would like to reproduce this once.

Reply:
> How can I reproduce the creation of these files?
> 
> I was not aware that apart from `.coverage`, there are more `.coverage*` files being created. I would like to reproduce this once.

Never mind. I found the answer to my question in your commit message. Thanks for writing the commit message in a detailed manner. I could indeed reproduce the creation of `.coverage.HOSTNAME.NUM.NUM` files.

So I am going to suggest that we update the glob pattern in `.gitignore` to `.coverage.*` so that it matches the one in `Makefile`.